### PR TITLE
fixed typo error.

### PR DESCRIPTION
**Pull request by J2P**
First of all, thank you for your contribution! :-)

Please makes sure that these checkboxes are checked before submitting your PR, thank you!

* [ ] Make sure that you propose PR to right branch: bugfix for `master`, feature for latest active branch `feature-x.x`.
* [ ] Make sure that you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
* [ ] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
* [ ] Rebase before creating a PR to keep commit history clear.
* [ ] Add some descriptions and refer relative issues for you PR.

Extra checklist:

**if** *isBugFix* **:**

  * [ ] Make sure that you add at least one unit test for the bug which you had fixed.

**elif** *isNewFeature* **:**

  * [ ] Update API docs for the component.
  * [ ] Update/Add demo to demonstrate new feature.
  * [ ] Update TypeScript definition for the component.
  * [ ] Add unit tests for the feature.

### CODE REVIEW

File: docs/react/use-with-create-react-app.en-US.md [hunk 1]
```diff
@@ -148,7 +148,7 @@ $ yarn add babel-plugin-import --dev
 Remove the `@import '~antd/dist/antd.css';` statement added before because `babel-plugin-import` will import styles and import components like below:
 
 ```diff
-  // scr/App.js
+  // src/App.js
```
Comment (yesmeck) [inline docs/react/use-with-create-react-app.en-US.md]: This is typo.

File: docs/react/use-with-create-react-app.en-US.md [hunk 2]
```diff
@@ -148,7 +148,7 @@ $ yarn add babel-plugin-import --dev
...
   import React, { Component } from 'react';
 - import Button from 'antd/lib/button';
 + import { Button } from 'antd';
```
Comment (yesmeck) [inline docs/react/use-with-create-react-app.en-US.md]: But this line is not type. 😄

Comment (J2P) [reply docs/react/use-with-create-react-app.en-US.md]: I did not change that line.
The original file has `-, +` symbols.

Comment (yesmeck) [reply docs/react/use-with-create-react-app.en-US.md]: Haha. My fault.

**Pull request merged**